src/dataDisplay/dataCards/dataCardColumn.jsx

```diff
@@ -37,6 +43,7 @@ function DataCardColumn(props) {
                 style={{
                     marginBottom: column.width ? '11px' : null,
                     width: column.width,
+                    ...column.style,
```

Comment:
Do we need to defensively check for `style`?
Maybe `...(style ?? {})` or something.

Reply:
Yeah, I thin we need to be defensive. Good catch.

Reply:
...

Reply:
Looks better now.  Resolving.